packages/react/src/auto/AutoForm.ts

```diff
@@ -120,6 +120,8 @@ export const useAutoForm = <
   // filter down the fields to render only what we want to render for this form
   const fields = useFormFields(metadata, props);
   const operatesWithRecordId = !!(metadata && isActionMetadata(metadata) && metadata.action.operatesWithRecordIdentity);
+  const isDeleteAction = metadata && isActionMetadata(metadata) && metadata.action.isDeleteAction;
+  const isGlobalAction = metadata && !isActionMetadata(metadata);
```

Comment:
How about `action.type === "globalAction"` like on line 129?

Reply:
Updated such that the check is `action.type === "globalAction"`